### ISSUE TITLE
Error 500 'NoneType' object is not subscriptable

### ISSUE THREAD

**Issue by zorun** [greenstatic/bigbluebutton-monitoring#4]
Hi,

Thanks for this simple monitoring app :)

With a very simple BBB instance (no Moodle integration) I get 500 errors that seem related to #1

```
ERROR in app: Exception on /api/meetings [GET]
Traceback (most recent call last):
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "bbb-mon/server.py", line 26, in api_meetings
    return jsonify(views.get_meetings())
  File "/home/bbbmon/bigbluebutton-monitoring/bbb-mon/views.py", line 53, in get_meetings
    origin_server = meeting['metadata']['bbb-origin-server-name']
TypeError: 'NoneType' object is not subscriptable
```

and the second one once I "fixed" the first one:

```
ERROR:server:Exception on /api/meetings [GET]
Traceback (most recent call last):
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/bbbmon/venv3/lib/python3.7/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "bbb-mon/server.py", line 26, in api_meetings
    return jsonify(views.get_meetings())
  File "/home/bbbmon/bigbluebutton-monitoring/bbb-mon/views.py", line 72, in get_meetings
    m['metadata']['origin-context'] = _bbb_context_convert_moodle(meeting['metadata']['bbb-context'])
TypeError: 'NoneType' object is not subscriptable
```

It seems that `meeting['metadata']` is None in my case. As a hack I set it to an empty dict instead and the 500 errors went away.

**Comment by greenstatic** [comment]
Thanks for the explanation and thorough bug report. I'll make a fix tomorrow / sunday at the latest.

**Comment by greenstatic** [comment]
fixed in v0.3.3
